modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -158,6 +158,46 @@ class Params<cv::gapi::Generic> {
     std::string m_tag;
 };
 
+// NB: Used by python wrapper
+// This class can be marked as SIMPLE, because it's implemented as pimpl
+class GAPI_EXPORTS_W_SIMPLE PyParams {
```

Comment:
Consider moving this into `misc/python/`

Reply:
But the header must be available for the python wrapper, `misc` isn't available, is it ?

Reply:
Main idea: need to exclude that from C++ users code.

Shadow files are processed by generator (but not included by cv2.cpp):
```
file(GLOB hdr ${OPENCV_MODULE_${m}_LOCATION}/misc/python/shadow*.hpp)
```

> modules/gapi/include/opencv2/gapi/infer/ie.hpp

Or move that into samething like modules/gapi/include/opencv2/gapi/infer/bindings_ie.hpp (which is not used in other headers)

Reply:
It also should be in *.so, second option is suitable

Reply:
> It also should be in *.so

In cv2.so only.
opencv_gapi.so is not necessary.

Implementation can be inlined.

Reply:
Fixed